src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Paint {
+    private Color color;
+    private String figureType;
+
+    public Figure(Color color) {
+        this.color = color;
+    }
+
+    public void setFigureType(String figureType) {
+        this.figureType = figureType;
+    }
+
+    public String getFigureType() {
+        return figureType;
+    }
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    @Override
+    public double area() {
+        return 0;
+    }
+
+    @Override
+    public void draw() {
+    }
```

Comment:
you don't need to override these methods in the abstract class, they will be considered default. And you have already redefined them in the successors